cfgmgr/teammgr.cpp

```diff
@@ -373,8 +375,33 @@ task_process_status TeamMgr::addLag(const string &alias, int min_links, bool fal
     string res;
 
     stringstream conf;
+
+    const string dump_path = "/var/warmboot/teamd/";
+    MacAddress mac_boot = m_mac;
+    /*set portchannel mac same with mac before warmStart ,when warmStart there is a file written by teamd*/
+    ifstream aliasfile(dump_path + alias);
+    if (WarmStart::isWarmStart() && aliasfile.is_open())
+    {
+# define PARTNER_SYSTEM_ID_OFFSET 40
+        string line;
+
+        while (getline(aliasfile, line))
+        {
+            ifstream memberfile(dump_path + line, ios::binary);
+            uint8_t mac_temp[ETHER_ADDR_LEN];
+
+            if (!memberfile.is_open())
+                continue;
+
+            memberfile.seekg(PARTNER_SYSTEM_ID_OFFSET, std::ios::beg);
+            memberfile.read((char *)mac_temp, ETHER_ADDR_LEN);
```

Comment:
instead of `(char *)` it's better to use `static_cast<char*>`

Reply:
Thanks, done